### PR TITLE
 Upgrade integrations SDK (NR-302384)

### PR DESCRIPTION
**Pull request by kang-makes**
Upgrade the integrations SDK removes the need to implement a storer in all integrations and rises the maximum interval to 5 minutes (if needed)

### CODE REVIEW

File: tests/integration/haproxy_test.go [hunk 1]
```diff
@@ -1,18 +1,20 @@
+//go:build integration
 // +build integration
```
Comment (sigilioso) [inline tests/integration/haproxy_test.go:2]: Can `+build integration` (the older format) be removed?

**Review by sigilioso — DISMISSED** [review]
LGTM

File: tests/integration/haproxy_test.go (hunk 1 above)
Comment (kang-makes) [reply tests/integration/haproxy_test.go]: Done in [#37999bb](https://github.com/newrelic/nri-haproxy/pull/157/commits/37999bb8b134681c3c764d25bac09a11b8b629c9)

**Review by sigilioso — APPROVED** [review]
LGTM

**Pull request merged**